app/extentions/Operators.scala

```diff
@@ -53,7 +53,7 @@ object Operators {
     import Results._
 
     def withUser(userId: UUID)(payload: User => Result)(implicit request: RequestWithUserData[AnyContent]): Result = {
```

Comment:
```suggestion
    def withUser(userId: UUID, includeDisabled: Boolean = false)(payload: User => Result)(implicit request: RequestWithUserData[AnyContent]): Result = {
```
Si on fait remonter, on n'est plus sûr de ne pas se plante, je pense

Reply:
C'est corrigé.